cpu/esp_common/Makefile.dep

```diff
@@ -93,3 +93,7 @@ ifneq (,$(filter esp_wifi_any,$(USEMODULE)))
   USEMODULE += netopt
   USEMODULE += ztimer_msec
 endif
+
+ifneq (,$(filter ztimer_msec,$(USEMODULE)))
+  USEMODULE += core_thread_flags
+endif
```

Comment:
```suggestion
ifneq (,$(filter esp_freertos_common,$(USEMODULE)))
  ifneq (,$(filter ztimer_msec,$(USEMODULE)))
    USEMODULE += core_thread_flags
  endif
endif
```

Reply:
Do you suggest it for better documentation? `esp_freertos_common` is always used.

Reply:
I still hope we can make it a standalone module one day, so then this would not be forgotten but just copied over.